### ISSUE TITLE
Failed to start synergys if display name contain spaces

### ISSUE THREAD

**Issue by ErdnussFlipS** [synergy/synergy#5153]
If your display name contains a space the command "synergys" fails.

See:
>"C:/Program Files/Synergy/synergys.exe" --name YourDisplayName With Spaces

When starting the server:
>DEBUG: started process, session=3, command="C:/Program Files/Synergy/synergys.exe" -f --no-tray --debug DEBUG --name Displayname with spaces --ipc --stop-on-desk-switch --enable-drag-drop --profile-dir "C:\Users\User1\AppData\Local" -c "C:/Users/User1/AppData/Local/Temp/qt_temp.Hp2572" --address :8080 --res-w 1920 --res-h 1080 --prm-wc 960 --prm-hc 540

**Comment by pinetum** [comment]
is it work?
```
"C:/Program Files/Synergy/synergys.exe" --name "YourDisplayName With Spaces"
```

**Comment by ErdnussFlipS** [comment]
Yes, of course. But this command is called by the synergy application and not by me from commandline.

**Comment by Pepedou** [comment]
This just happened to me.
